internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,27 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+// fillClusterIDFromMigrationReq fills the clusterID for the passed in monitors.
+func fillClusterIDFromMigrationReq(parameters map[string]string) (string, error) {
+	var err error
+	var rclusterID string
+	mons := parameters["monitors"]
+	for _, m := range strings.Split(mons, ",") {
+		rclusterID, err = util.GetClusterIDFromMon(m)
+		if err != nil {
+			if (strings.Contains(err.Error(), util.ErrConfFileFetchFailed)) ||
+				(strings.Contains(err.Error(), util.ErrUnmarshalFailed)) {
+				return "", err
```

Comment:
you can check ` "missing configuration of cluster ID for mon %q", mon`(deing const in errors.go) and return error 
```suggestion
if err != nil && !errors.Is(err, util.ErrorMissingConfiguration){
return "",err
}
```

Reply:
Done.. @Madhu-1